USER_MANUAL.md

```diff
@@ -340,7 +352,163 @@ _TODO_
 
-Content for the "Program Inputs" section goes here.
+### &lt;parameter_level&gt; (programs)
+
+**Data Type:** String
+
+**Default input:** 'programs'
+
+**Description:** A string indicating the parameters in file are at the program settings level
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** Must be set to ```parameter_level: programs``` for an output setting parameter file.
+
+### &lt;version&gt; (programs)
+
+**Data type:** String
+
+**Default input:** 4.0
+
+**Description:** Specify version of LDAR-Sim. See section _[Versioning of Parameter Files](#versioning-of-parameter-files)_ for more information.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Improper versioning will prevent simulator from executing.
+
+### &lt;program_name&gt;
+
+**Data Type:** String
+
+**Default input:** "default"
+
+**Description:** The name of the program. Typical naming convention is to include "P_" before a name.
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** Each program must have a unique program name. If names are duplicated, they will override each other.
+
+### &lt;method_labels&gt;
+
+**Data Type:** List[strings]
+
+**Default input:** []
+
+**Description:** A list of the methods used within the program. For example, the following will use the aircraft and the OGI_FU methods:
+
+```yaml
+method_labels:
+- aircraft
+- OGI_FU
+```
+
+**Notes on acquisition:** N/A
+
+**Notes of caution:** The method labels that are referenced must be identical to an existing [method_name](#method_name), it is case sensitive.
+
+### &lt;economics&gt;
+
+**Description** Economic values that are used to generate cost related figures.
+
+**Note:** The following economics parameters are currently placeholders and will be reimplemented in a future patch.
+
+#### &lt;carbon_price_tonnes_CO2_equivalent&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 65.0
+
+**Description:** The federal price on carbon in Canada of $65/tonne CO2e (as of April 2023) is input as a default metric to compare the cost to mitigation ratios of LDAR programs to.
+
+**Notes of acquisition:** The current fuel charge rates for Canada can be found [here](https://www.canada.ca/en/revenue-agency/services/forms-publications/publications/fcrates/fuel-charge-rates.html).
+
+**Notes of caution:**  This rate rises annually and should be updated in the model to reflect changes in the price.
+
+#### &lt;global_warming_potential_CH4&gt; WIP
+
+**Data Type:** Numeric
+
+**Default input:** 28.0
+
+**Description:** GWP of 28 over a 100-year time period was chosen as a default input. The model uses this value to convert between CH4 and CO2e when required. This value can be changed to 84-86 over 20 years to explore the impact that GWP has on mitigation costs.
+
+**Notes of acquisition:** This value is from [Chapter 8](https://www.ipcc.ch/site/assets/uploads/2018/02/WG1AR5_Chapter08_FINAL.pdf) in the IPCC's Assessment Report 5 from Working Group 1 page 714, 2018.
```

Comment:
Update this?

Reply:
28 is still valid.

Reply:
Is there a newer reference we can use?

Reply:
I couldn't find any neatly laid out tables like the reference we have in IPCC AR6, which is why I think keeping this reference is fine.
I did also find a quote on page 6 (122) of https://www.ipcc.ch/report/ar6/syr/downloads/report/IPCC_AR6_SYR_AnnexesIndex.pdf 
"... parties have agreed to use GWP100 values from the IPCC AR5 or
GWP100 values from a subsequent IPCC Assessment Report to report..."